docs/UserGuide.adoc

```diff
@@ -72,6 +72,11 @@ Examples:
 Shows a list of all persons, along with their non-private details, in the address book. +
 Format: `list`
 
+== Sort all persons inside the address book : `sort`
+
+Sorts all persons in the address book and shows a sorted list. +
```

Comment:
You may consider specifying that instead of sorting the person in the address book, it only shows a sorted list.